### ISSUE TITLE
Readthedocs?

### ISSUE THREAD

**Issue by bmcfee** [mir-evaluation/mir_eval#241]
Any interest in setting mir_eval up on rtd?  It would be nice to have automatic doc builds for PRs, if nothing else.

**Comment by craffel** [comment]
Sounds like a good idea to me.

**Comment by bmcfee** [comment]
Good project for [docathon](https://bids.github.io/docathon/) this coming week.  I'll take a crack at it probably around wednesday.

**Comment by bmcfee** [comment]
Hrm, looks like only the project owner can set up the integration. :cry:

**Comment by bmcfee** [comment]
Blocked by #375

**Comment by bmcfee** [comment]
RTD integration is now working, but the sphinx configs need some work to get autodoc running again.

**Comment by stefan-balke** [comment]
Is this one-page layout up for debate or could we organize it in topics? IMHO, would be easier to get to the information needed. However, CTRL+F one a single page is also charming in a way.

What you think?

**Comment by bmcfee** [comment]
I would absolutely be in favor of moving to a modern layout and RTD theme.  I think our docs are verbose enough that a one-pager is pretty unwieldy.

**Comment by bmcfee** [comment]
Ok, I think we're in business: https://mir-eval.readthedocs.io/latest/

Closing this out.  Final step will be to modify our gh-pages to point to the rtd build instead.

**Comment by stefan-balke** [comment]
Looks good, thanks for picking this up!
